src/main/java/frc/robot/lib/interfaces/CTREConfigs.java

```diff
@@ -60,25 +56,24 @@ public CTREConfigs() {
         swerveCanCoderConfig.sensorTimeBase = SensorTimeBase.PerSecond;
 
         /* Arm motors PIDF config values */
-        RobotMap.shoulderMotor.config_kF(0, SHOULDER_MOTOR.F, 10);
-        RobotMap.shoulderMotor.config_kP(0, SHOULDER_MOTOR.P, 10);
-        RobotMap.shoulderMotor.config_kI(0, SHOULDER_MOTOR.I, 10);
-        RobotMap.shoulderMotor.config_kD(0, SHOULDER_MOTOR.D, 10);
+        RobotMap.leftShoulderMotor.config_kF(0, Constants.LEFT_SHOULDER_MOTOR.F, 10);
+        RobotMap.leftShoulderMotor.config_kP(0, Constants.LEFT_SHOULDER_MOTOR.P, 10);
+        RobotMap.leftShoulderMotor.config_kI(0, Constants.LEFT_SHOULDER_MOTOR.I, 10);
+        RobotMap.leftShoulderMotor.config_kD(0, Constants.LEFT_SHOULDER_MOTOR.D, 10);
 
-        RobotMap.elbowMotor.config_kF(0, ELBOW_MOTOR.F, 10);
-        RobotMap.elbowMotor.config_kP(0, ELBOW_MOTOR.P, 10);
-        RobotMap.elbowMotor.config_kI(0, ELBOW_MOTOR.I, 10);
-        RobotMap.elbowMotor.config_kD(0, ELBOW_MOTOR.D, 10);
+        RobotMap.leftElbowMotor.config_kF(0, Constants.LEFT_ELBOW_MOTOR.F, 10);
+        RobotMap.leftElbowMotor.config_kP(0, Constants.LEFT_ELBOW_MOTOR.P, 10);
+        RobotMap.leftElbowMotor.config_kI(0, Constants.LEFT_ELBOW_MOTOR.I, 10);
+        RobotMap.leftElbowMotor.config_kD(0, Constants.LEFT_ELBOW_MOTOR.D, 10);
 
-        RobotMap.wristMotor.config_kF(0, WRIST_MOTOR.F, 10);
-        RobotMap.wristMotor.config_kP(0, WRIST_MOTOR.P, 10);
-        RobotMap.wristMotor.config_kI(0, WRIST_MOTOR.I, 10);
-        RobotMap.wristMotor.config_kD(0, WRIST_MOTOR.D, 10);
+        RobotMap.rightShoulderMotor.config_kF(0, Constants.RIGHT_SHOULDER_MOTOR.F, 10);
```

Comment:
The PID values will be the same for both motors that are controlling the same joint.